recipes/matplotlib-venn/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.11.4" %}
+package:
+  name: matplotlib-venn
+  version: {{ version }}
+
+source:
+  fn: matplotlib-venn-{{ version }}.zip
+  url: https://pypi.io/packages/source/m/matplotlib-venn/matplotlib-venn-{{ version }}.zip
+  md5: 55264984346687371e446b1f8d27a420
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - matplotlib
```

Comment:
You probably don't need to list `matplotlib`, `numpy`, and `scipy` at build here.